### PR TITLE
Docs add try with jupyter lite

### PR DESCRIPTION
**Pull request by HaudinFlorence**
Add a try with JupyterLite in ipyleaflet documentation

### CODE REVIEW

File: docs/environment.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies:
   - branca
   - shapely
   - pydata-sphinx-theme
+  - jupyterlite-sphinx
```
Comment (martinRenou) [inline docs/environment.yml:18]: You can only install jupyterlite-sphinx with pip for now

File: docs/source/jupyterlite_config.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "LiteBuildConfig": {
+      "federated_extensions": [
+        "https://github.com/conda-forge/releases/releases/download/noarch/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2",
+        "https://github.com/conda-forge/releases/releases/download/noarch/ipycanvas-0.11.0-pyhd8ed1ab_0.tar.bz2/ipycanvas-0.11.0-pyhd8ed1ab_0.tar.bz2"
```
Comment (martinRenou) [inline docs/source/jupyterlite_config.json:5]: You probably need to remove ipycanvas from this file and point to ipyleaflet wheel and conda package instead

**Review by martinRenou — CHANGES_REQUESTED** [review]

**Comment by HaudinFlorence** [comment]
Add a try with JupyterLite in ipyleaflet documentation

File: docs/environment.yml [hunk 3]
```diff
@@ -15,7 +15,8 @@ dependencies:
   - branca
   - shapely
   - pydata-sphinx-theme
+
   - pip:
     - ..
-
-
+    - ipyleaflet
```
Comment (martinRenou) [inline docs/environment.yml:21]: We don't need this, we already install ipyleaflet with `..`

File: docs/source/jupyterlite_config.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+    "LiteBuildConfig": {
+      "federated_extensions": [
+        "https://github.com/conda-forge/releases/releases/download/noarch/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2",
+        "https://anaconda.org/conda-forge/ipyleaflet/0.15.0/download/noarch/ipyleaflet-0.15.0-pyhd8ed1ab_3.tar.bz2"
```
Comment (martinRenou) [inline docs/source/jupyterlite_config.json:5]:
```suggestion
        "https://github.com/conda-forge/releases/releases/download/noarch/ipyleaflet-0.15.0-pyhd8ed1ab_3.tar.bz2/ipyleaflet-0.15.0-pyhd8ed1ab_3.tar.bz2"
```

File: docs/source/jupyterlite_config.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      ],
+      "ignore_sys_prefix": true,
+      "piplite_urls": [
+        "https://files.pythonhosted.org/packages/py2.py3/i/ipycanvas/ipycanvas-0.11.0-py2.py3-none-any.whl",
```
Comment (martinRenou) [inline docs/source/jupyterlite_config.json:9]:
```suggestion
        "https://files.pythonhosted.org/packages/py2.py3/i/ipyleaflet/ipyleaflet-0.15.0-py2.py3-none-any.whl",
```

File: docs/source/ipyleaflet.ipynb [hunk 6]
```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```
Comment (martinRenou) [inline docs/source/ipyleaflet.ipynb:5]: Can you please clean the cell outputs of the Notebook?

**Review by martinRenou — CHANGES_REQUESTED** [review]

File: docs/source/jupyterlite_config.json (hunk 4 above)
Comment (martinRenou) [reply docs/source/jupyterlite_config.json:5]: Maybe that will work better?

**Review by martinRenou — APPROVED** [review]

**Comment by martinRenou** [comment]
Thanks a lot! Looks great :)

**Pull request merged**